queries/vala/highlights.scm

```diff
@@ -0,0 +1,387 @@
+; Pointers
+
+(address_of_identifier
+  "&" @symbol
+  (_)*
```

Comment:
this has no effect

Reply:
![Difference](https://user-images.githubusercontent.com/48367298/151662735-0f43a5cc-5a92-479a-af6e-ab3ac3435387.png)

The intention is to highlight the address symbol differently from the and operator

Reply:
but `(_)*` is not needed for that. `(_)*` mean zero or more anything which does not add any information to the query.